packages/components/icons/src/icons.visualroute.jsx

```diff
@@ -61,82 +61,80 @@ const renderIcon = (iconName, color, size) => {
 
 export const component = ({ themes }) => (
   <Switch>
+    <Route path={routePath} exact>
+      <ul>
+        {colors.map((color) => (
+          <li key={`${routePath}/${color}`}>
+            <a href={`${routePath}/${color}`}>{`${routePath}/${color}`}</a>
+          </li>
+        ))}
+        <li>
+          <a href={`${routePath}/theme`}>{`${routePath}/theme`}</a>
+        </li>
+        <li>
+          <a href={`${routePath}/inline-svg`}>{`${routePath}/inline-svg`}</a>
+        </li>
+      </ul>
+    </Route>
```

Comment:
I added these links to the subpages as the `/icons` page does not actually render anything.